docs/consume-packages/Package-Restore.md

```diff
@@ -162,6 +162,14 @@ To restore packages listed in the project file with PackageReference, use the th
    ```
 
    Make sure that the MSBuild output indicates that the build completed successfully.
+   
+> [!Note]
+> msbuild has a `-restore` switch which will run `Restore` and then build. See [Restoring and building with one MSBuild command](/nuget/reference/msbuild-targets#restoring-and-building-with-one-msbuild-command).
```

Comment:
It's more than just running restore and the build, it reloads certain things, meaning `-restore` is different to `-t:restore,build` or `-t:restore -t:build`.

Reply:
@zivkan - can you suggest updates to address your feedback? Use the "suggestions" feature :) 
![image](https://user-images.githubusercontent.com/16904420/77799986-2aaf5c80-7033-11ea-935f-5c4256381f4a.png)

Reply:
```suggestion
> msbuild has a `-restore` switch which will run `Restore`, reload the project, and then build. See [Restoring and building with one MSBuild command](/nuget/reference/msbuild-targets#restoring-and-building-with-one-msbuild-command).
```